dist/watchFiles.js

```diff
@@ -0,0 +1,62 @@
+"use strict";
```

Comment:
Don't include `dist` files in source control, remove this and gitignore this folder.